assets/src/components/developers/RunKit.ts

```diff
@@ -88,18 +88,22 @@ async function handler({event, payload}) {
 
 async function handleMessageCreated(message) {
   const {body, customer_id, conversation_id} = message;
-  const isCustomerMessage = !!customer_id;
 
-  if (isCustomerMessage && body.toLowerCase().startsWith('test')) {
+    messageBody = body.toLowerCase()
```

Comment:
the indentation here looks a bit inconsistent with the `const {..` above

also looks like `messageBody` is a global variable? should probably be `const messageBody`

Reply:
might be worth just pasting all this code in a js file and running prettier on it so the formatting is ok